.github/workflows/auto-update-gemfiles.yml

```diff
@@ -0,0 +1,169 @@
+        run: ruby -v
+
+      - name: Bundle
+        run: bundle install
+
+      - name: Update latest
+        run: bundle exec rake edge:latest
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: 'datadog-gem-${{ matrix.engine.name }}-${{ matrix.engine.version }}-gha${{ github.run_id }}'
+          path: gemfiles/${{ matrix.engine.name }}_${{ matrix.engine.version }}_*
+          retention-days: 3
+
+  aggregate:
+    needs: build
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read # read secrets
+      pull-requests: write
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download artifacts for Ruby 2.5
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-2.5-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 2.6
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-2.6-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 2.7
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-2.7-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 3.0
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-3.0-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 3.1
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-3.1-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 3.2
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-3.2-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 3.3
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-3.3-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for Ruby 3.4
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-ruby-3.4-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for JRuby 9.2
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-jruby-9.2-gha${{ github.run_id }}'
+          path: gemfiles
+      
+      - name: Download artifacts for JRuby 9.3
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-jruby-9.3-gha${{ github.run_id }}'
+          path: gemfiles
+
+      - name: Download artifacts for JRuby 9.4
+        uses: actions/download-artifact@v4
+        with:
+          name: 'datadog-gem-jruby-9.4-gha${{ github.run_id }}'
+          path: gemfiles
+      
+      # ADD NEW RUBIES HERE
+      
+      - run: git diff
+
+      - name: Create Pull Request
```

Comment:
Since we are having a fixed branch name, it would be helpful to clarify with a comment

> The default behaviour is to create a pull request that will be continually updated with new changes until it is merged or closed. 

Otherwise, the behaviour becomes unclear after one week from now without merging the PR.